src/command.cpp

```diff
@@ -48,6 +48,57 @@ const char* Command::getstring() const
 }
 
 // add opcode
+bool Command::checkError(int64_t& e, JSON& json, ErrorDetails& errorDetails)
```

Comment:
The comment above is for the next method

Reply:
Fixed